amundsen_application/api/__init__.py

```diff
@@ -19,5 +20,10 @@ def index(path: str) -> Any:
     return render_template("index.html", env=ENVIRONMENT)  # pragma: no cover
 
 
+def storybook(path: str) -> Any:
+    ## TODO: Stuck here, not finding the template
+    return render_template("../storybook-static/templates/index.html", env=ENVIRONMENT)  # pragma: no cover
```

Comment:
I haven't been able to correctly build everything yet to check this out (messaged on Slack with my error), but one thing I can see is that we have things configured so that the app is looking for templates in `amundsen_application/static/templates`. So I imagine we'd create a new template in that directory.

Reply:
I see. I will search for a way to have multiple template folders then, as it is not possible to move the one of storybook there without creating a big mess.
Thanks!